ScoutSuite/providers/azure/rules/rulesets/default.json

```diff
@@ -291,6 +291,12 @@
                 "level": "warning"
             }
         ],
+        "storageaccount-encrypted-not-customer-managed.json": [
+            {
+                "enabled": true,
+                "level": "danger"
```

Comment:
```suggestion
                "level": "warning"
```

Reply:
According to the CISv1.2.0 documentation, this rule has a profile applicability of level 2. Shouldn't this translate to a level of "Danger"?